docs/whats-new/2019-12.md

```diff
@@ -57,6 +57,10 @@ Welcome to what's new in .NET docs for December 2019. This article lists some of
 
 - [Version tolerant serialization](../standard/serialization/version-tolerant-serialization.md) - Fix duplicate link in see also, and cleanup article
 
+## .NET APIs
+
+.NET API documentation was updated to reflect .NET Core 3.1 release.
```

Comment:
Should this include a link to the API browser, with the 3.1 release chosen: https://docs.microsoft.com/en-us/dotnet/api/?view=netcore-3.1

Reply:
Yes, good point.